### ISSUE TITLE
MXD initial `terraform apply` throws error

### ISSUE THREAD

**Issue by anschl** [eclipse-tractusx/tutorial-resources#268]
### Current Behavior
Error while running Terraform:
```
kubernetes_job.seed_connectors_via_mgmt_api: Still creating... [40s elapsed]
kubernetes_job.seed_connectors_via_mgmt_api: Still creating... [50s elapsed]
kubernetes_job.seed_connectors_via_mgmt_api: Still creating... [1m0s elapsed]
╷
│ Error: job: default/seed-connectors is not in complete state
│ 
│   with kubernetes_job.seed_connectors_via_mgmt_api,
│   on seed_data.tf line 24, in resource "kubernetes_job" "seed_connectors_via_mgmt_api":
│   24: resource "kubernetes_job" "seed_connectors_via_mgmt_api" {
```

### Expected Behavior
Kubernetes cluster gets fully initialized

### Steps To Reproduce
Follow the instructions in https://github.com/eclipse-tractusx/tutorial-resources/blob/main/mxd/README.md until `terraform apply` throws the error.
Running `terraform apply` again results in the same error.
<details>
<summary>Running `kubectl logs job/seed-connectors` while Terraform is deploying: </summary>

```
Defaulted container "newman-bob" out of: newman-bob, newman-alice, newman-miw
newman

MXD Management API Seed

❏ SeedData
↳ Create Business Partner Group
  POST http://10.96.144.238:8081/management/business-partner-groups [409 Conflict, 234B, 44ms]
  ✓  Status code is 204 (if new BPN) or 409 Conflict (if BPN already exists)

↳ Create Asset 1
  POST http://10.96.144.238:8081/management/v3/assets [409 Conflict, 217B, 8ms]
  ✓  Status code is 204 No Content (if new asset) or 409 Conflict (if asset already exists)

↳ Create Asset 2
  POST http://10.96.144.238:8081/management/v3/assets [409 Conflict, 217B, 8ms]
  ✓  Status code is 204 No Content (if new asset) or 409 Conflict (if asset already exists)

↳ Create Policy
  POST http://10.96.144.238:8081/management/v2/policydefinitions [400 Bad Request, 1.01kB, 7ms]
  1. Status code is 204 No Content (if new policy) or 409 Conflict (if policy already exists)

↳ Create Contract Definition 1
  POST http://10.96.144.238:8081/management/v2/contractdefinitions [409 Conflict, 231B, 7ms]
  ✓  Status code is 204 No Content (if new contract definition) or 409 Conflict (if contract deifinition already exists)

↳ Create Contract Definition 2
  POST http://10.96.144.238:8081/management/v2/contractdefinitions [409 Conflict, 231B, 7ms]
  ✓  Status code is 204 No Content (if new contract definition) or 409 Conflict (if contract deifinition already exists)

┌─────────────────────────┬──────────────────┬──────────────────┐
│                         │         executed │           failed │
├─────────────────────────┼──────────────────┼──────────────────┤
│              iterations │                1 │                0 │
├─────────────────────────┼──────────────────┼──────────────────┤
│                requests │                6 │                0 │
├─────────────────────────┼──────────────────┼──────────────────┤
│            test-scripts │               18 │                0 │
├─────────────────────────┼──────────────────┼──────────────────┤
│      prerequest-scripts │               12 │                0 │
├─────────────────────────┼──────────────────┼──────────────────┤
│              assertions │                6 │                1 │
├─────────────────────────┴──────────────────┴──────────────────┤
│ total run duration: 299ms                                     │
├───────────────────────────────────────────────────────────────┤
│ total data received: 1.45kB (approx)                          │
├───────────────────────────────────────────────────────────────┤
│ average response time: 13ms [min: 7ms, max: 44ms, s.d.: 13ms] │
└───────────────────────────────────────────────────────────────┘

  #  failure         detail                                                                                   
                                                                                                              
 1.  AssertionError  Status code is 204 No Content (if new policy) or 409 Conflict (if policy already exists) 
                     expected 400 to be one of [ 200, 204, 409 ]                                              
                     at assertion:0 in test-script                                                            
                     inside "SeedData / Create Policy"
```
</details>

**Comment by hemantxpatel** [comment]
Hi @anschl, thanks for reporting.
This issue is currently not reproducible. I just tried and it works fine. Could you please check if you have latest code and you are on `main` branch.

**Comment by paullatzelsperger** [comment]
@hemantxpatel i think this may have been caused by a recent upstream EDC change. Can you try to remove the `odrl:or` wrapping object like so: 
```json
{
  "@context": {
    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "PolicyDefinitionRequestDto",
  "@id": "1",
  "policy": {
    "@type": "odrl:Set",
    "odrl:permission": [
      {
        "odrl:action": "USE",
        "odrl:constraint": {
          "@type": "LogicalConstraint",
          "odrl:or": [
            {
              "@type": "Constraint",
              "odrl:leftOperand": "{{TX_NAMESPACE}}BusinessPartnerGroup",
              "odrl:operator": {
                "@id": "odrl:eq"
              },
              "odrl:rightOperand": "{{BUSINESS_PARTNER_GROUP}}"
            }
          ]
        }
      }
    ]
  }
}
```

to 

```json
{
  "@context": {
    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "PolicyDefinitionRequestDto",
  "@id": "1",
  "policy": {
    "@type": "odrl:Set",
    "odrl:permission": [
      {
        "odrl:action": "USE",
        "odrl:constraint": {
          "@type": "LogicalConstraint",
          "odrl:leftOperand": "{{TX_NAMESPACE}}BusinessPartnerGroup",
          "odrl:operator": {
            "@id": "odrl:eq"
          },
          "odrl:rightOperand": "{{BUSINESS_PARTNER_GROUP}}"
        }
      }
    ]
  }
}
```

**Comment by hemantxpatel** [comment]
Hi @paullatzelsperger, Yes I verified existing policy fails with latest EDC version, and your updated policy definition works fine.

For now, we have fixed EDC `0.6.0` version into MXD and both policy definitions works fine with this version.

**Comment by hemantxpatel** [comment]
@paullatzelsperger with the latest EDC, I just changed the existing policy definition slightly, and it worked.
I just changed `"@type": "LogicalConstraint",` -> `"@type": "odrl:LogicalConstraint",` inside constraint object.
